src/si/search.js

```diff
@@ -65,28 +66,23 @@ const searchAll = (term = '', opts = {}) => {
       term = opts.term
     }
 
-    let page = 1
-    let results = []
-    let tmpData = []
-    let _continue = true
-
-    while (_continue && page <= 15) {
-      // We stop at page === 15 because nyaa.si offers a maximum of 1000 results
-      // which means 14 pages of 75 results with the last one containing only 25.
-      try {
-        results = _.concat(results, tmpData)
+    try {
+      const { results: fResults, maxPage } = await searchPage(term, 1, opts, true)
+      const searchs = []
+      for (let page = 2; page <= maxPage; page++) {
```

Comment:
I'm a fervent defender of `++var` instead of `var++` 😄

Reply:
xD